src/site/stages/build/drupal/graphql/GetAllPages.graphql.js

```diff
@@ -11,12 +13,14 @@ module.exports = `
   ${fragments}
   ${landingPage}
   ${page}
+  ${healthCareRegionPage}
 
   query GetAllPages {
     nodeQuery(limit: 500) {
       entities {
         ... landingPage
         ... page
+        ... healthCareRegionPage
```

Comment:
Should this go in the other query files, too?

Reply:
makes sense.